config/initializers/logging.rb

```diff
@@ -0,0 +1,20 @@
+config = Rails.application.config
+
+if ENV["RAILS_LOG_TO_STDOUT"].present?
+  # heroku and docker: dump everything to stdout
+  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+elsif ENV["RAILS_LOG_TO_SYSLOG"]
```

Comment:
yes, but '0'.present? is also true so we should adjust this and RAILS_LOG_TO_STDOUT I guess ...